packages/lib-classifier/src/components/Classifier/components/Layout/components/DefaultLayout/DefaultLayout.js

```diff
@@ -12,7 +12,8 @@ import QuickTalk from '@components/Classifier/components/QuickTalk'
 import SubjectViewer from '@components/Classifier/components/SubjectViewer'
 import TaskArea from '@components/Classifier/components/TaskArea'
 
-const columnSize = pxToRem(380)
+const subjectSize = pxToRem(680)
+const taskSize = pxToRem(380)
```

Comment:
I'm wondering if we need this calculation, or if it would be better to explicitly state the sizes here.
```suggestion
const subjectSize = '45rem'
const taskSize = '25rem'
```

Reply:
`pxToRem` just divides the provided number by 15, but 15px isn't the base font size so the columns here don't come out as 680px and 380px wide in the rendered page.
https://github.com/zooniverse/front-end-monorepo/blob/a1af82bf7da492fa3b004fec31e37cb221242dbf/packages/lib-react-components/src/helpers/pxToRem.js#L1-L14

Reply:
The rendered grid column widths, in Firefox, are 729px and 405px on my laptop, which corresponds to a base font size of ~16px.

Reply:
I would have thought the base font size would be [14px based on the theme](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-grommet-theme/src/index.js#L473) but I think you're right, it looks like it's ~16px. Explicitly stating the sizes makes sense to me.

Reply:
The theme sets 14px for paragraphs but the base font size will be the font size that’s set for the page body.